README.md

```diff
@@ -67,6 +67,8 @@ Here is overview of availability of DNS entries for GPG keys:
 | OpenSuse | :heavy_check_mark:          | :heavy_check_mark: |
 | RPM Fusion | [requested](https://bugzilla.rpmfusion.org/show_bug.cgi?id=5927)        | :x: |
 | Dropbox    | requested        | :x: |
+| Debian (source) | :heavy_check_mark:        | :x: |
+| Ubuntu (source) | :heavy_check_mark:        | :x: |
```

Comment:
```
$ resolvectl openpgp debian-release@lists.debian.org
68606aa4798144c50621db9dc332a71a329a49664a0dcaecf36f7b5c._openpgpkey.lists.debian.org: resolve call failed: '68606aa4798144c50621db9dc332a71a329a49664a0dcaecf36f7b5c._openpgpkey.lists.debian.org' not found
```
Does not seem to exist. Or am I missing something?

Reply:
well if the code source code you have there it is an example but it is functional as I tell you it is ext which will make the basic debian packages

try this its already give you the base code

that I took and adapt

```
sudo apt-get -y debhelper cdbs lintian build-essential fakeroot devscripts dh-make dput coreutils wget

wget https://github.com/xsuchy/distribution-gpg-keys/archive/refs/tags/distribution-gpg-keys-1.72-1.tar.gz -O distribution-gpg-keys_1.72.orig.tar.gz

tar -xvf distribution-gpg-keys_1.72.orig.tar.gz

cd distribution-gpg-keys*/

dh_make -e youraddress@email.tld -s -y
```

after for the keys you are looking for I do not even see where you found them

the most important thing is the debian/rules file

after the rest the packagers will adapt themselves